cmd/sidecars/network-passt-binding/domain/configurator.go

```diff
@@ -32,18 +32,21 @@ import (
 	"kubevirt.io/kubevirt/pkg/network/istio"
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/device"
 
+	netdriver "kubevirt.io/kubevirt/pkg/network/driver"
 	"kubevirt.io/kubevirt/pkg/network/namescheme"
 	"kubevirt.io/kubevirt/pkg/network/vmispec"
 )
 
 type NetworkConfiguratorOptions struct {
 	IstioProxyInjectionEnabled bool
 	UseVirtioTransitional      bool
+	NetworkHandler             netdriver.NetworkHandler
```

Comment:
This generic handler should be marked as deprecated.
Please use the relevant specific driver, e.g. [netlink](https://github.com/kubevirt/kubevirt/tree/main/pkg/network/driver/netlink).

You should be able to use a simple interface to allow unit tests (you do not need the fake).

Reply:
ahh ok, I will use that.

Reply:
What do you mean with simple interface ?

Reply:
Ok done.

Reply:
I think you just did it.